scripts/components/api-changes-validator/import_statements_renderer.ts

```diff
@@ -0,0 +1,58 @@
+import {
+  NamespaceDefinitions,
+  UsageStatementsGeneratorOutput,
+} from './types.js';
+import { EOL } from 'os';
+
+/**
+ * Renders import statements.
+ */
+export class ImportStatementsRenderer {
+  /**
+   * Creates import statements renderer.
+   */
+  constructor(
+    private readonly generatorOutputs: Array<UsageStatementsGeneratorOutput>,
+    private readonly namespaceDefinitions: NamespaceDefinitions,
+    private readonly packageName: string
+  ) {}
+  render = (): string => {
+    const importStatements: Array<string> = [];
+    for (const generatorOutput of this.generatorOutputs) {
+      if (generatorOutput.importStatement) {
+        importStatements.push(generatorOutput.importStatement);
+      } else if (generatorOutput.symbolDescriptor) {
+        if (
+          !this.namespaceDefinitions.namespaceBySymbol.has(
+            generatorOutput.symbolDescriptor.symbolName
+          )
+        ) {
+          importStatements.push(
+            `import { ${generatorOutput.symbolDescriptor.symbolName} } from '${this.packageName}';`
+          );
+        }
+      }
+    }
+
+    for (const namespaceName of this.namespaceDefinitions.topLevelNamespaces) {
+      if (namespaceName.startsWith('__export__')) {
```

Comment:
Could you help me fine where we add `__export__`?

Reply:
See `Out of scope` section in description.

The change that will follow looks like that https://github.com/aws-amplify/amplify-backend/pull/1869 .

We also included this case in tests here https://github.com/aws-amplify/amplify-backend/blob/bf9de828fa68c29ef516e9c19362057fb0305572/scripts/components/api-changes-validator/test-resources/test-projects/without-breaks/project-with-namespace/API.md
( the API.md in that test project is simulating how `index.internal.ts` would be translated into api extract)